src/error.rs

```diff
@@ -201,10 +203,15 @@ impl fmt::Display for Error {
 			Self::LiquidityFeeTooHigh => {
 				write!(f, "The given operation failed due to the LSP's required opening fee being too high.")
 			},
-			Self::CannotRbfFundingTransaction => write!(f, "Cannot RBF a channel funding transaction."),
+			Self::CannotRbfFundingTransaction => {
+				write!(f, "Cannot RBF a channel funding transaction.")
+			},
 			Self::TransactionNotFound => write!(f, "The transaction was not found in the wallet."),
-			Self::TransactionAlreadyConfirmed => write!(f, "The transaction is already confirmed and cannot be modified."),
-			Self::NoSpendableOutputs => write!(f, "The transaction has no spendable outputs.")
+			Self::TransactionAlreadyConfirmed => {
+				write!(f, "The transaction is already confirmed and cannot be modified.")
+			},
+			Self::NoSpendableOutputs => write!(f, "The transaction has no spendable outputs."),
```

Comment:
I know it was added a while back, but still. When can transaction have no spendable outputs?

Reply:
When creating a transaction it's possible that some or all of the UTXO's passed in are used in lightning funding transactions. In which case the intended transaction does not have any spendable outputs.